README.md

```diff
@@ -38,7 +38,18 @@ To start/stop a service, run `systemctl start <name>` or `systemctl stop <name>`
 
 Create the `update-operator` deployment and `update-agent` daemonset.
 
+For non-RBAC cluster:
```

Comment:
We don't really have to encourage those still running non-RBAC clusters. Who are these people? (half joking)

Reply:
Agreed